src/components/Header/HeaderLogo.js

```diff
@@ -10,24 +10,7 @@ function HeaderLogo() {
         align-items: center;
       `}
     >
-      <img
-        alt=""
-        src={headerLogoSvg}
-        width={4 * GU}
-        css={`
-          margin-right: ${1 * GU}px;
-        `}
-      />
-      <h1
-        css={`
-          display: flex;
-          height: 100%;
-          align-items: center;
-          ${textStyle('body1')};
-        `}
-      >
-        Aragon Court
-      </h1>
+      <img alt="" src={headerLogoSvg} width={17.5 * GU} />
```

Comment:
Similar to the sanity check above, this appears to be quite a lot larger than before :)

Reply:
Same as above, we exported the icons with the integrated background.